s3/pkg/datastore/yig/config/config.go

```diff
@@ -135,21 +135,32 @@ func (cc *CacheConfig) Parse(vals map[string]interface{}) error {
 }
 
 type DatabaseConfig struct {
-	DbType     int
-	DbUrl      string
-	DbPassword string
+	DbType       string
+	DbUrl        string
+	DbPassword   string
+	MaxIdleConns int
+	MaxOpenConns int
 }
 
 func (dc *DatabaseConfig) Parse(vals map[string]interface{}) error {
+	dc.MaxIdleConns = 0
+	dc.MaxOpenConns = 1024
```

Comment:
why so many ?

Reply:
Thanks @rhsakarpos for your review, because if there is so high work load, the more max open conns  will reduce the latency between the yig storage backend and the tidb.